release_notes/index.adoc

```diff
@@ -20,3 +20,11 @@ technical notes, see the
 https://docs.openshift.com/enterprise/3.1/release_notes/index.html[OpenShift
 Enterprise 3.1 Release Notes].
 endif::[]
+
+[[release-versioning-policy]]
+== Versioning Policy
+{product-title} does not have non-backwards compatible changes and, therefore,
```

Comment:
I'm not against your explanation. But what is the reason to have at least 3 minor releases? Should we mention bugfix releases anywhere (3.2.1 for example)?
Wouldn't it be easier to just adopt http://semver.org ? I don't see it having stricter requirement from what is written above. Bumping version for non-functionality reasons would still be compatible with semver I think. At least would not break user expectations too much.